src/base32.app.src

```diff
@@ -1,10 +1,5 @@
 {application, base32, [
-    {description, "Erlang implementation of base32 encoding and decoding"},
-    {vsn, "0.1"},
-    {registered, []},
-    {applications, [
-        kernel,
-        stdlib
-    ]},
-    {env, []}
+    {description, "Erlang implementation of base32 encoding and decoding."},
+    {vsn, git},
+    {applications, [kernel, stdlib]}
 ]}.
```

Comment:
hex is going to complain if you don't add a license and links here (and also for good practice). As in https://github.com/dnsimple/dns_erlang/blob/713ddec0fa5606d444a415960dfe645b13856759/src/dns_erlang.app.src#L9-L10

Reply:
Done.